user/helm-chart/values-kubernets.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Prefix used for defining KubeStellar objects in the cluster
+prefix: kubestellar
```

Comment:
It seems a bit redundant to have the word "kubestellar" in both an object's name and in its namespace.
Does not have to be fixed in this PR.

Reply:
There is a chance for deployment in a different ns with other objects in it... for the moment, I will keep the prefix